app/assets/stylesheets/global/_base.scss

```diff
@@ -42,6 +56,22 @@ input[type="submit"] {
   border: 0.1em solid $color_bg-dark;
 }
 
+.active {
```

Comment:
This treatment of the `active` class is specific to the `#page_container` and should not be global. If we ever have anything else that uses an `active` class (which is likely) this will conflict. This class should be nested within `#page_container`.

Reply:
Nesting active into `#page_container` breaks the functionality for unknown reasons, but web inspector suggests it isn't getting assigned the new properties despite getting classed.